test/exec/child.js

```diff
@@ -1,3 +1,5 @@
 // does nothing child process
 console.log("Child process.id: "+process.pid);
 console.log(" - - - - - - - - - - - - - - - - - - - - - - - ");
+// keep process alive for one minute
+setTimeout(function () {}, 60000);
```

Comment:
@jedwards1211 please clarify why this timeout is required and why it _has_ to be for 60 sec. thanks!